pallets/profile/src/lib.rs

```diff
@@ -81,8 +81,8 @@ pub mod weights;
 #[frame_support::pallet]
 pub mod pallet {
 	use frame_support::{dispatch::DispatchResult,
-	storage::bounded_vec::BoundedVec,
-	pallet_prelude::*};
+						storage::bounded_vec::BoundedVec,
```

Comment:
What is happening with the spacing here?

Reply:
y Thats why I normally just use Rustfmt on save. Tell me what settings you use for formatting and I will replicate. Manually re-formatting now.

Reply:
Resolved in next commit.